src/main/java/Duke.java

```diff
@@ -1,10 +1,44 @@
+import java.util.Scanner;
 public class Duke {
+    public static void printSplit(){
+        System.out.println("    ✻❊✽✼❉✱✲✾❃❋❈❆✿❀❁");
+    }
+
+
+    public static void printBye() {
+        printSplit();
+        System.out.println("    Bye. Hope to see you again soon!");
+        printSplit();
+    }
+
     public static void main(String[] args) {
+        Task task = new Task();
+        Duke split = new Duke();
```

Comment:
I noticed the Duke split object is only used here. I was wondering what is the reasoning behind it? or if it could maybe be replaced by another line?